pkg/apiserver/api_installer.go

```diff
@@ -204,10 +204,11 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	}
 	versionedStatus := indirectArbitraryPointer(versionedStatusPtr)
 	var (
-		getOptions     runtime.Object
-		getOptionsKind string
-		getSubpath     bool
-		getSubpathKey  string
+		getOptions          runtime.Object
+		versionedGetOptions runtime.Object
+		getOptionsKind      string
+		getSubpath          bool
+		getSubpathKey       string
 	)
 	if isGetterWithOptions {
 		getOptions, getSubpath, getSubpathKey = getterWithOptions.NewGetOptions()
```

Comment:
Sorry I cant find where is this NewGetOptions() method implemented?
https://github.com/GoogleCloudPlatform/kubernetes/search?utf8=%E2%9C%93&q=NewGetOptions returns the implementation for PodLogOptions only.
I am wondering why does getOptions not have any of these parameters?